### PR TITLE
Refactor detected tokens layout

### PR DESCRIPTION
**Pull request by tewshi**
## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/asset-manager/newly-detected/NewlyDetectedAssetTable.vue [hunk 1]
```diff
@@ -0,0 +1,249 @@
+<script setup lang="ts">
+import { type ComputedRef, type Ref } from 'vue';
+import { Blockchain } from '@rotki/common/lib/blockchain';
+import { type NewDetectedToken } from '@/types/websocket-messages';
+import type {
+  DataTableColumn,
+  DataTableSortColumn
+} from '@rotki/ui-library-compat/dist/components';
```
Comment (kelsos) [inline frontend/app/src/components/asset-manager/newly-detected/NewlyDetectedAssetTable.vue:8]: Don't we export them from elsewhere?

**Comment by kelsos** [comment]
Code looks fine, I left a question for the imports only.

I will do a final check in the morning and then we can merge.

File: frontend/app/src/components/asset-manager/newly-detected/NewlyDetectedAssetTable.vue (hunk 1 above)
Comment (tewshi) [reply frontend/app/src/components/asset-manager/newly-detected/NewlyDetectedAssetTable.vue:8]: auto imports, should be exported via `@rotki/ui-library-compat`

**Review by lukicenturi — APPROVED** [review]
LGTM!

**Review by kelsos — APPROVED** [review]
lgtm @tewshi thanks

**Pull request merged**